stacklet/client/sinistral/cli.py

```diff
@@ -168,4 +168,9 @@ def login(ctx, username, password, *args, **kwargs):
 
 
 if __name__ == "__main__":
-    cli()
+    try:
+        cli()
+    except Exception:
+        import pdb, sys, traceback
+        traceback.print_exc()
+        pdb.post_mortem(sys.exc_info()[-1])
```

Comment:
This seems helpful for debugging, but do we want it as the default behavior for end-users? Maybe a `--debug-errors` flag?

Reply:
Output from the CI / Lint job is oddly unhelpful, but it looks like it's all from this block.

black is complaining about wanting an extra blank line after the import, flake8 is complaining about the multi-import, and debug is complaing that pdb is imported at all.

Reply:
end users go through cli entry points, this one only triggers if doing python -m stacklet.client.sinistral.cli